pkg/api/config.go

```diff
@@ -11,6 +11,8 @@ import (
 type Config struct {
 	// ClusterVersion defines release version of the plugin used to build the cluster
 	ClusterVersion string `json:"clusterVersion,omitempty"`
+	// ClusterLogLevel specifies the OpenShift component log level
+	ClusterLogLevel uint8 `json:"clusterLogLevel,omitempty"`
```

Comment:
This is going back on what I said to you before, but it's little extra work: LogLevel would be better as a struct containing different components and their log levels.  Example: apiserver, controllers, node.  Can use an int rather than a uint8.

Reply:
yes, otherwise you get all or nothing.

Reply:
good idea. I'll get this in